projects/klippa/ngx-enhancy-forms/src/lib/form/form-validation-error/form-validation-error.ts

```diff
@@ -0,0 +1,36 @@
+import { InjectionToken } from '@angular/core';
+
+export class FormValidationError extends Error {
+	_path: string;
+	_message: string;
+
+	constructor(path: string, message: string) {
+		super(message);
+		this.name = 'FORM_VALIDATION_ERROR';
```

Comment:
since the name is always `FORM_VALIDATION_ERROR `, can just be a static final constant?

Reply:
As in a constant in the file? `name` is defined by the error interface, I can't make it `static readonly` in the class as that conflicts with the definition in the super class.